js/validateinput.js

```diff
@@ -76,7 +77,28 @@ form.addEventListener('submit', function handleSubmit(event) {
   });
 
   if (isValid && emailValid) {
-    form.submit();
+    try {
+      const response = await fetch('https://formspree.io/f/xzblbbzd', {
+        method: 'POST',
+        body: formData,
+        headers: {
+          Accept: 'application/json',
+        },
+      });
+
+      if (!response.ok) {
+        responseMessage.textContent = 'Failed to submit the form. Please try again.';
+        responseMessage.classList.add('error-message');
+      } else {
+        responseMessage.textContent = 'Form submitted successfully!';
```

Comment:
After the form is successfully submitted, let's `reset` the form and clear the `localstorage`.